apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerDbType.cs

```diff
@@ -316,6 +317,10 @@ public static SpannerDbType FromClrType(System.Type type)
             {
                 return Bytes;
             }
+            if (type == typeof(SpannerDate))
```

Comment:
Was this just missing before? (It doesn't seem related to the custom conversion part.)

Reply:
Yes Jon. It should have gone with SpannerDate changes.